ext/mysqli/tests/mysqli_auth_pam.phpt

```diff
@@ -19,8 +19,8 @@ if (!$res = $link->query("SHOW PLUGINS"))
 
 $have_pam = false;
 while ($row = $res->fetch_assoc()) {
-    if (isset($row['Name']) && ('mysql_clear_password' == $row['Name'])) {
-        $have_pam = true;
+    if (isset($row['Name']) && in_array($row['Name'], array('pam', 'mysql_clear_text', 'auth_pam_compat'))) {
```

Comment:
Is the change from mysql_clear_password to mysql_clear_text intentional here? As far as I can tell, the plugin is called mysql_clear_password, even though various related options use the "cleartext" terminology instead.

Reply:
nope. That was a mistake. Thanks @nikic . Force pushing now.